app/src/organisms/ProtocolSetup/RunSetupCard/CollapsibleStep.tsx

```diff
@@ -31,7 +38,9 @@ export function CollapsibleStep({
   label,
   toggleExpanded,
   children,
+  calibrationComplete,
 }: CollapsibleStepProps): JSX.Element {
+  const { t } = useTranslation(['protocol_setup'])
```

Comment:
```suggestion
  const { t } = useTranslation('protocol_setup')
```

The hook also accepts a single string `namespace` instead of a `namespace[]`. Which we can use because we're only loading the `'protocol_setup'` namespace here.